src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
```

Comment:
what happens when you print orders that you fetched in `orderDao.getAllByUser()` method? can you see user?

Reply:
I print all information about orders, like this example 
`Order{id=1, tickets=[Ticket{id=1, movieSession=MovieSession{id=1, movie=Movie{id=1, title='Fast and Furious', description='An action film about street racing, heists, and spies.'}, cinemaHall=CinemaHall{id=1, capacity=100, description='first hall with capacity 100'}, showTime=2022-01-05T10:55:24.812772}, user=User{id=1}}], orderDate=2022-01-04T10:55:24.990662, user=User{id=1}}`